src/crypto/tls/handshake_server.go

```diff
@@ -142,7 +142,7 @@ func (c *Conn) readClientHello() (*clientHelloMsg, error) {
 	// or GetCertifciate(). Hence, it is not currently possible to reject ECH if
 	// we don't recognize the inner SNI. This may or may not be desirable in the
 	// future.
-	clientHello, err = c.echAcceptOrReject(clientHello)
+	clientHello, err = c.echAcceptOrReject(clientHello, false /* afterHRR */)
```

Comment:
I think is better to move the comment outside the function.

Reply:
Done (here and in handshake_server_tls13.go).